docs/operation: Reword K8s API auth management (#1960)
scality/metalk8s: docs/operation/account_administration.rst

   credentials.

Administering MetalK8s GUI, Kubernetes API and Salt API
*******************************************************

> During installation, MetalK8s configures Kubernetes API to accept Basic Auth

Reviewer: ```suggestion
During installation, MetalK8s configures the Kubernetes API to accept Basic
```

> authentication method, with default credentials ``admin`` / ``admin``.

Reviewer: ```suggestion
authentication, with default credentials ``admin`` / ``admin``.
```

Services exposed by MetalK8s, such as
:ref:`its GUI <quickstart-services-admin-ui>` or

> :ref:`Salt API <quickstart-services-salt>`, rely on Kubernetes API for

Reviewer: ```suggestion
:ref:`Salt API <quickstart-services-salt>`, rely on the Kubernetes API for
```

authenticating their users. As such, changing the credentials for

> Kubernetes API default user will also change the credentials required to

Reviewer: ```suggestion
authenticating their users. As such, changing the credentials of a
Kubernetes API user will also change the credentials required to
```

connect to either one of these services.

Managing Kubernetes API username and password
---------------------------------------------

  .. warning::

     The procedures mentioned below must be carried out on every control-plane

>      :term:`Node`, or more specifically, a Node bearing the

Reviewer: ```suggestion
     :term:`Node`, or more specifically, any Node bearing the
```

     ``node-role.kubernetes.io/master`` label.

> #. Edit the credentials file located at ``/etc/kubernetes/htpasswd``, replacing

Author: Do we want to give a better explanation on this file (e.g. give a reference to [the docs](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#static-password-file))? Or, since we plan on using OIDC in the near future (2.4.1 maybe?), should we just stick with this for now?

Reviewer: Stick with this and burn HTTP auth with fire ASAP
